reward/staking_manager.go

```diff
@@ -117,7 +136,11 @@ func GetStakingInfo(blockNum uint64) *StakingInfo {
 	// Calculate staking info from block header and updates it to cache and db
 	calcStakingInfo, _ := updateStakingInfo(stakingBlockNumber)
 	if calcStakingInfo == nil {
+		if stakingManager.migrationCheck.InMigration() {
+			logger.Error("YOU MUST NOT RESTART NODE", "action", "if you want to restart the node, you should wait a day after the migration completed", "reason", "failed to store staking info while migration")
```

Comment:
This error message is supposing the failure of updating DB. However, there are other failures returning `nil` calcStakingInfo. For more precise messages, we need to check the error of `updateStakingInfo`.

Reply:
Thank you for your comment. I reflected to be more readable as a system admin.

Reply:
I reflected your opinion. Thank you :)